src/Avalonia.Base/Rendering/Composition/Server/ServerCompositionTarget.cs

```diff
@@ -190,14 +190,21 @@ public void Render()
                     }
                 }
 
+                targetContext.PushClip(_dirtyRect);
                 targetContext.Clear(Colors.Transparent);
                 targetContext.Transform = Matrix.Identity;
                 if (_layer.CanBlit)
+                {
                     _layer.Blit(targetContext);
```

Comment:
Are you sure that this codepath isn't being chosen on windows? it uses glBlitFramebuffer internally which completely skips skia.

Reply:
Yes, blit is fine. The Clear operation above is not that much. I wonder if it makes sense to Clear it before Blit since Blit will overwrite it anyway.